packages/react-components/src/components/VideoTile.tsx

```diff
@@ -229,6 +261,27 @@ export const VideoTile = (props: VideoTileProps): JSX.Element => {
   const canShowLabel = showLabel && (displayName || (showMuteIndicator && isMuted));
   const participantStateString = participantStateStringTrampoline(props, locale);
 
+  /* @conditional-compile-remove(pinned-participants) */
+  const optionsMenuItems = (menuItems: VideoTileMenuItems): IContextualMenuItem[] => {
+    const contextualMenuItems: IContextualMenuItem[] = [];
+    menuItems.forEach((item: VideoTileMenuItem) => {
+      contextualMenuItems.push({
+        key: item.key,
```

Comment:
why are we not setting the aria label here?

Reply:
Has Alex given us a A11y spec for the feature yet? if we dont want to have a special label for this we should leave it blank. The narrator should announce the text for the item on its own in its ordering.